Mathlib/AlgebraicGeometry/Morphisms/QuasiSeparated.lean

```diff
@@ -169,9 +169,8 @@ theorem quasiSeparated_respectsIso : MorphismProperty.RespectsIso @QuasiSeparate
   quasiSeparated_eq_diagonal_is_quasiCompact.symm ▸ quasiCompact_respectsIso.diagonal
 #align algebraic_geometry.quasi_separated_respects_iso AlgebraicGeometry.quasiSeparated_respectsIso
 
-open List in
 theorem QuasiSeparated.affine_openCover_TFAE {X Y : Scheme.{u}} (f : X ⟶ Y) :
-    TFAE
+    List.TFAE
```

Comment:
Back when this was ported, I complained that I didn't really like the optics of `List.TFAE` and prefer the `TFAE` spelling. That's also what we had in ml3.

I would prefer to have `open List` or maybe `open List (TFAE)` at the top of the file.